packages/payload/src/query-presets/constraints.ts

```diff
@@ -74,7 +74,7 @@ export const getConstraints = (config: Config): Field => ({
                 },
               ],
             },
-            relationTo: 'users',
+            relationTo: config.admin?.user ?? 'users',
```

Comment:
No need to fallback here as Payload does this automatically during config sanitization.

```suggestion
            relationTo: config.admin?.user
```